innopoints/core/image.py

```diff
@@ -0,0 +1,47 @@
+"""Image manipulation."""
+
+from typing import Dict
+
+from PIL import Image
+
+from innopoints.core.helpers import abort
+
+
+def crop(image: Image.Image, dimensions: Dict[str, str]):
+    """Crop an image to the dimensions specified with keys 'x', 'y', 'width', 'height'."""
+    if 'x' not in dimensions:
```

Comment:
I don't understand, why the check for `x` explicitly, when you check for the rest of the keys in the try/catch?

Reply:
In some cases the cropping isn't desirable, so no keys are passed, it checks for this case

Reply:
but this doesn't check for the absence of all keys, just `x`. What if everything else was passed except for it?

Reply:
Then it's an invalid crop anyway

Reply:
ok, so it shouldn't be silently ignored. It should be caught by the `except KeyError` that will rethrow an error